Client/WarFare/UIHelp.h

```diff
@@ -10,6 +10,7 @@
 #endif // _MSC_VER > 1000
 
 #include <N3Base/N3UIBase.h>
+#include <N3Base/N3UIButton.h>
```

Comment:
Don't include this in the header. Include it in the source file.
There's no need to expose it to anything including it.

These are forward-declared already.